openfl/interface/collaborator_manager.py

```diff
@@ -66,7 +68,17 @@ def create(collaborator_manager_path):
                     collaborator_manager_path / 'shard_config.yaml')
     shutil.copyfile(SITEPACKS / 'openfl/component/collaborator_manager/shard_descriptor.py',
                     collaborator_manager_path / 'shard_descriptor.py')
-
+    url = 'http://api.github.com/repos/smallstep/cli/releases/latest'
+    download_step_bin(url, 'step_linux', 'amd', prefix=collaborator_manager_path)
```

Comment:
It also seems like we are now tied to linux

Reply:
There are step binaries for windows also/